target/linux/ramips/dts/mt7621_hilink_hlk-7621a.dts

```diff
@@ -0,0 +1,78 @@
+// SPDX-License-Identifier: GPL-2.0
+
+#include "mt7621.dtsi"
+
+/ {
+	compatible = "hilink,hlk-7621a", "mediatek,mt7621-soc";
+	model = "HLK-7621A";
```

Comment:
"HiLink ..."

Reply:
> "HiLink ..."

@adschm Isn't it lowercase here?

Reply:
Lower case for the compatible, but the vendor is missing at all for the model.